src/oauth.js

```diff
@@ -56,29 +56,29 @@ ret.oauth = function(platform, userAddress, scopes) {
     goTo('https://www.dropbox.com/1/oauth2/authorize');
   } else if (platform === 'googledrive') {
     goTo('https://accounts.google.com/o/oauth2/auth');
-  } else if (platform === 'remotestorage' || platform === 'remotestorage-no-https') {
-    var parts = userAddress.split('@');
-    var prefix = 'https://';
-    if (platform === 'remotestorage-no-https') {
-      prefix = 'http://';
-    }
-    requestJSON(prefix + parts[1] + '/.well-known/webfinger?resource='+encodeURIComponent('acct:'+userAddress),
-        undefined, function(err, data) {
```

Comment:
I like how here you can see directly that `requestJSON` is about requesting JSON, and that the `data` variable is exactly the contents of the webfinger document. This simplicity makes everything easy to debug.